perftrackerlib/client.py

```diff
@@ -344,6 +344,12 @@ def __init__(self, tag, uuid1=None, group=None, binary=None, cmdline=None, descr
 
         self.validate()
 
+    def __eq__(self, other):
+        assert isinstance(other, ptTest)
+        attributes = ["tag", "group", "category", "metrics", "less_better", "errors", "warnings", "status"]
```

Comment:
The 'errors', 'warnings' and 'status' are not needed, this is 'outcome', while 'tag', 'group', 'category', 'metrics' and 'less_better' are the 'input parameters'.

Reply:
This was fixed